view/src/components/account.js

```diff
@@ -0,0 +1,332 @@
+			[event.target.name]: event.target.value
+		});
+	};
+
+	handleImageChange = (event) => {
+		this.setState({
+			image: event.target.files[0]
+		});
+	};
+
+	profilePictureHandler = (event) => {
+		event.preventDefault();
+		this.setState({
+			uiLoading: true
+		});
+		authMiddleWare(this.props.history);
+		const authToken = localStorage.getItem('AuthToken');
+		let form_data = new FormData();
+		form_data.append('image', this.state.image);
+		form_data.append('content', this.state.content);
+		axios.defaults.headers.common = { Authorization: `${authToken}` };
+		axios
+			.post('/user/image', form_data, {
+				headers: {
+					'content-type': 'multipart/form-data'
+				}
+			})
+			.then(() => {
+				window.location.reload();
+			})
+			.catch((error) => {
+				if (error.response.status === 403) {
+					this.props.history.push('/login');
+				}
+				console.log(error);
+				this.setState({
+					uiLoading: false,
+					imageError: 'Error in posting the data'
+				});
+			});
+	};
+
+	updateFormValues = (event) => {
+		event.preventDefault();
+		this.setState({ buttonLoading: true });
+		authMiddleWare(this.props.history);
+		const authToken = localStorage.getItem('AuthToken');
+		axios.defaults.headers.common = { Authorization: `${authToken}` };
+		const formRequest = {
+			firstName: this.state.firstName,
+			lastName: this.state.lastName,
+			country: this.state.country
+		};
+		axios
+			.post('/user', formRequest)
+			.then(() => {
+				this.setState({ buttonLoading: false });
+			})
+			.catch((error) => {
+				if (error.response.status === 403) {
+					this.props.history.push('/login');
+				}
+				console.log(error);
+				this.setState({
+					buttonLoading: false
+				});
+			});
+	};
+
+	render() {
+		const { classes, ...rest } = this.props;
+		if (this.state.uiLoading === true) {
+			return (
+				<main className={classes.content}>
+					<div className={classes.toolbar} />
+					{this.state.uiLoading && <CircularProgress size={150} className={classes.uiProgess} />}
+				</main>
+			);
+		} else {
+			return (
+				<main className={classes.content}>
+					<div className={classes.toolbar} />
+					<Card {...rest} className={clsx(classes.root, classes)}>
+						<CardContent>
+							<div className={classes.details}>
+								<div>
+									<Typography className={classes.locationText} gutterBottom variant="h4">
+										{this.state.firstName} {this.state.lastName}
+									</Typography>
+									<Button
+										variant="outlined"
+										color="primary"
+										type="submit"
+										size="small"
+										startIcon={<CloudUploadIcon />}
+										className={classes.uploadButton}
+										onClick={this.profilePictureHandler}
+									>
+										Upload Photo
+									</Button>
+									<input type="file" onChange={this.handleImageChange} />
+
+									{this.state.imageError ? (
+										<div className={classes.customError}>
+											{' '}
+											Wrong Image Format || Supported Format are PNG and JPG
```

Comment:
This is almost impossible to review due to the deep nesting with such wide tabs; it wraps before the first non-whitespace character is even shown.  Please convert this to spaces, preferably two per indentation due to the depth of nesting required here.